packages/react-components/babel-preset-storybook-full-source/src/index.js

```diff
@@ -0,0 +1,11 @@
+// This is internal code and should be a dev dependency
```

Comment:
lets remove `index.js` and `index.d.ts` to `index.dev.js/index.dev.d.ts`  and update paths. with that we can require(`index`) within both `index.dev.js` and `delcaration file`. As a follow up work this can be leveraged to actually be able to run `build` on this in production mode to not ship by accident those helpers we are using for overriding node resolution mechanism.